external/radiation/radiation/radlw/radlw_main.py

```diff
@@ -85,14 +88,29 @@ def mcica_subcol(iovrlw, cldf, nlay, dz, de_lgth, iplon, rand2d):
                 else:
                     cdfunc[n, k] = cdfunc[n, k] * tem1
 
+    if iovrlw == 3:  # decorrelation length overlap
+
+        fac_lcf = np.exp(-0.5 * (dz[1:] + dz[:-1]) / de_lgth)
```

Comment:
Could add a comment that this follows directly from [here](https://github.com/ai2cm/fv3gfs-fortran/blob/2a3ea739723d6152fd03d419c76d25741f7fa9da/FV3/gfsphysics/physics/radlw_main.f#L1967)